Firebase/Core/FIRLogger.m

```diff
@@ -104,11 +118,15 @@ void FIRLoggerInitializeASL() {
       sFIRLoggerDebugMode = NO;
     }
 
-    // Need to call asl_add_output_file so that the logs can appear in Xcode's console view. Set
-    // the ASL filter mask for this output file up to debug level so that all messages are
-    // viewable in the console.
-    asl_add_output_file(sFIRLoggerClient, STDERR_FILENO, kFIRLoggerCustomASLMessageFormat,
-                        ASL_TIME_FMT_LCL, ASL_FILTER_MASK_UPTO(ASL_LEVEL_DEBUG), ASL_ENCODE_SAFE);
+#if TARGET_OS_SIMULATOR
+    // Need to call asl_add_output_file so that the logs can appear in Xcode's console view when
+    // running iOS 7. Set the ASL filter mask for this output file up to debug level so that all
+    // messages are viewable in the console.
+    if (majorOSVersion == 7) {
```

Comment:
Did you test on majorOSVersion 8,9,10 simulators, since this block would have been executed for them before?